assets/js/guides.js

```diff
@@ -14,6 +14,13 @@ $(document).ready(function () {
     return element;
   };
 
+  //Move Outline to top of the position of first guide card
+  const startOutlinePosition = function () {
+    const firstGuideCardTop = $('.guide-card').first().offset().top;
+
+    $(".js-scroll-with-user").offset({top: firstGuideCardTop});
```

Comment:
Should the initial alignment not be done with CSS rather than JS? Otherwise, it'll jump on page load.

Reply:
Nice point. I will do this in CSS.